src/FSharpVSPowerTools.Core/PrintfSpecifiersUsageGetter.fs

```diff
@@ -0,0 +1,46 @@
+﻿module FSharpVSPowerTools.PrintfSpecifiersUsageGetter
+
+open Microsoft.FSharp.Compiler
+open FSharpVSPowerTools.UntypedAstUtils
+
+[<NoComparison>]
+type PrintfSpecifierUse =
+    { SpecifierRange: Range.range
+      ArgumentRange: Range.range }
+
+let private sortRange (r: Range.range) = r.StartLine, r.StartColumn
```

Comment:
Does this mean 'startPos' i.e. a tuple denoting start position? (sortRange sounds like a totally different thing.)